packages/plugin-essentials/sources/commands/plugin/remove.ts

```diff
@@ -0,0 +1,63 @@
+import {BaseCommand}                                                                   from '@yarnpkg/cli';
+import {Configuration,  MessageName,  Project,  StreamReport, FormatType, structUtils} from '@yarnpkg/core';
+import {PortablePath, ppath, xfs}                                                      from '@yarnpkg/fslib';
+import {Command, Usage, UsageError}                                                    from 'clipanion';
+
+// eslint-disable-next-line arca/no-default-export
+export default class PluginRemoveCommand extends BaseCommand {
+  @Command.String()
+  name!: string;
+
+  static usage: Usage = Command.Usage({
+    category: `Plugin-related commands`,
+    description: `remove a plugin`,
+    details: `
+      This command deletes the specified plugin from the .yarn/plugins folder and removes it from the configuration.
+
+      **Note:** The plugins have to be referenced by their name property, which can be obtained using the \`yarn plugin runtime\` command. Shorthands are not allowed.
```

Comment:
We'll probably want to support shorthands imo, otherwise it'll feel weird to do `yarn plugin import exec` but not be able to do `yarn plugin remove exec` 🤔